Project.toml

```diff
@@ -9,6 +9,7 @@ FastGaussQuadrature = "442a2c76-b920-505d-bb47-c5924d526838"
 ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 LinearSolve = "7ed4a6bd-45f5-4d41-b270-4a48e9bafcae"
+NearestNeighbors = "b8a86587-4115-5ab1-83bc-aa920d37bbce"
```

Comment:
If I rebember correctly, the [compat] entry is mandatory for all dependencies if we want to register Bcube. Can you please add it?